### PR TITLE
feat: set admin hostname on web container as alternative dns

### PR DESCRIPTION
**Pull request by klyse**

### CODE REVIEW

**Comment by burnedikt** [comment]
Pretty sure the same also needs to happen for customization... Or in general any service that calls another one directly using its Hostname

File: tasks/discovery_portal.yml [hunk 1]
```diff
@@ -86,3 +86,6 @@
     network_mode: default
     networks:
       - name: "{{ network_names.main }}"
+        aliases:
+          - redis
```
Comment (burnedikt) [inline tasks/discovery_portal.yml:90]:
```suggestion
```

Portal doesn't need to be accessible as `redis` I guess

File: tasks/customization_service.yml [hunk 2]
```diff
@@ -96,6 +96,9 @@
     network_mode: default
     networks:
       - name: "{{ network_names.main }}"
+        aliases:
+          - redis
```
Comment (burnedikt) [inline tasks/customization_service.yml:100]:
```suggestion
```

Customization doesn't need to be accessible as redis I guess

**Review by burnedikt — APPROVED** [review]

**Review by burnedikt — CHANGES_REQUESTED** [review]
`-redis` shouldn't be an alias

**Comment by klyse** [comment]
> `-redis` shouldn't be an alias

crap, that was a copy paste issue 😢

**Review by burnedikt — APPROVED** [review]

**Pull request merged**